data/json/items/generic.json

```diff
@@ -2939,5 +2939,19 @@
     "color": "green",
     "use_action": "DISASSEMBLE",
     "flags": [ "NO_REPAIR" ]
+  },
+  {
+    "id": "plastic_six_dice",
```

Comment:
I believe there is already a die item, I don't think we need another

Reply:
There are RPG die and also metal RPG die. Not really a part of a gambling though.

Reply:
Yeah, iirc no 6 sided dice